README.md

```diff
@@ -1,68 +1,119 @@
-# Flask + React + Postgres Starter 
+# Team Standup App  
 
-This is a minimal sample Flask and React starter code that demonstrates how both frameworks can be used together in a single page web Application.
+This is a simple single page web app that integrates the Flask and React framework. It uses Track 2 versions of the [Azure SDK for Python](https://github.com/Azure/azure-sdk-for-python), specifically [azure-cosmos](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/cosmos/azure-cosmos) and [azure-keyvault-secrets](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/keyvault/azure-keyvault-secrets), in addition to [azure-identity](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/identity/azure-identity) for authentication purposes. 
 
-The code is based on https://github.com/dternyak/React-Redux-Flask and https://github.com/creativetimofficial/material-dashboard-react
 
-## Tutorial
+The code is based on https://github.com/dternyak/React-Redux-Flask and https://github.com/creativetimofficial/material-dashboard-react.
 
-## 1. Setting Up The Project
+## How to Run
+
+## 1. Setting up Azure Services
+First, sign up for a free [Azure](https://azure.microsoft.com/en-us/free/) account if you don't already have one. Sign into https://portal.azure.com.
+
+[Create a resource group](https://github.com/lilyjma/azurethings/blob/master/createResourceGroup.md) to store the resources that you'll be using here--Azure Cosmos DB and Key vault. Then follow the instructions in the links below to create each resource:
+
+(Remember to store them in the resource group you created; this will make it easier to clean up the resources in the future.)
+
+1. [Azure Cosmos DB](https://docs.microsoft.com/en-us/azure/cosmos-db/create-cosmosdb-resources-portal#create-an-azure-cosmos-db-account)
```

Comment:
I like to include the CLI commands along side the portal instructures or a section which is 

## Azure Portal
1. Portal steps here
## Azure CLI
1. CLI steps here

Reply:
The CLI commands are also done executed on Portal though--on the Cloud Shell. You could also execute CLI commands on your local machine but that requires some extra configuration so I thought it'd be the easiest to just do it on Cloud Shell.